src/plugins/intel_npu/tests/functional/shared_tests_instances/behavior/ov_plugin/internal_properties_tests.cpp

```diff
@@ -0,0 +1,184 @@
+
+            std::string expect_value = property_item.second.as<std::string>();
+            std::string actual_value = actual_property_value.as<std::string>();
+            EXPECT_EQ(actual_value, expect_value) << "Property is changed in wrong way";
+        }
+    }
+}
+
+const std::vector<ov::AnyMap> CorrectPluginMutableProperties = {
+    {{ov::internal::exclusive_async_requests.name(), true}},
+    {{ov::intel_npu::dpu_groups.name(), 1}},
+    {{ov::intel_npu::dma_engines.name(), 1}},
+    {{ov::intel_npu::compilation_mode.name(), "DefaultHW"}},
+    {{ov::intel_npu::compilation_mode_params.name(), "dump-task-stats=false propagate-quant-dequant=0"}},
+    {{ov::intel_npu::platform.name(),
+      removeDeviceNameOnlyID(
+          ov::test::utils::getTestsDeviceNameFromEnvironmentOr(std::string(ov::intel_npu::Platform::AUTO_DETECT)))}},
+    {{ov::intel_npu::stepping.name(), 0}},
+    {{ov::intel_npu::max_tiles.name(), 2}},
+    {{ov::intel_npu::use_elf_compiler_backend.name(), ov::intel_npu::ElfCompilerBackend::NO}},
+    {{ov::intel_npu::profiling_type.name(), ov::intel_npu::ProfilingType::INFER}}};
+
+const std::vector<ov::AnyMap> IncorrectMutablePropertiesWrongValueTypes = {
+    {{ov::intel_npu::compilation_mode.name(), -3.6}},
+    {{ov::intel_npu::compiler_type.name(), ov::intel_npu::ElfCompilerBackend::NO}},
+    {{ov::intel_npu::stepping.name(), "V1"}},
+    {{ov::intel_npu::max_tiles.name(), "two"}},
+    {{ov::intel_npu::use_elf_compiler_backend.name(), "N"}},
+    {{ov::intel_npu::profiling_type.name(), 10}},
+    {{ov::intel_npu::tiles.name(), "none"}},
+    {{ov::intel_npu::dma_engines.name(), false}},
+    {{ov::intel_npu::compilation_mode_params.name(), "not-a-param=true"}},
+};
+
+INSTANTIATE_TEST_SUITE_P(smoke_BehaviorTests,
+                         OVPropertiesTestsNPU,
+                         ::testing::Combine(::testing::Values(ov::test::utils::DEVICE_NPU),
+                                            ::testing::ValuesIn(CorrectPluginMutableProperties)),
+                         (ov::test::utils::appendPlatformTypeTestName<OVPropertiesTestsNPU>));
+
+INSTANTIATE_TEST_SUITE_P(smoke_BehaviorTests,
+                         OVPropertiesIncorrectTestsNPU,
+                         ::testing::Combine(::testing::Values(ov::test::utils::DEVICE_NPU),
+                                            ::testing::ValuesIn(IncorrectMutablePropertiesWrongValueTypes)),
+                         (ov::test::utils::appendPlatformTypeTestName<OVPropertiesIncorrectTestsNPU>));
+
+INSTANTIATE_TEST_SUITE_P(smoke_BehaviorTests,
+                         OVCheckSetSupportedRWMetricsPropsTestsNPU,
+                         ::testing::Combine(::testing::Values(ov::test::utils::DEVICE_NPU),
+                                            ::testing::ValuesIn(CorrectPluginMutableProperties)),
+                         (ov::test::utils::appendPlatformTypeTestName<OVCheckSetSupportedRWMetricsPropsTestsNPU>));
+
+}  // namespace ov::test::behavior
```

Comment:
new line

Reply:
Thanks for pointing out. I added the modifications